### ISSUE TITLE
Which DSDT will work on this laptop?

### ISSUE THREAD

**Issue by Mtkhan13** [chris1111/HP-Probook-EliteBook-Package-Creator#21]
Hi,
Please guide me which EliteBook or Probook Model's DSDT will work on this machine?
https://drivertechs.wordpress.com/2014/10/14/shabaz-sharif-laptop-drivers-i3-hp-1000/

**Comment by chris1111** [comment]
you can try the 4530s; Intel HD 3000 is not support on Mojave and Catalina,  you have to select patch HD 3000

**Comment by Mtkhan13** [comment]
Hi, thanks for reply,I tried this and also selected the 3000 patch but it stuck at boot screen. 
-- 
Sent from my Android phone with mail.com Mail. Please excuse my brevity.On 7/3/19, 6:18 AM chris1111 <notifications@github.com> wrote:
you can try the 4530s; Intel HD 3000 is not support on Mojave and Catalina,  you have to select patch HD 3000

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/issues/21?email_source=notifications\u0026email_token=AFMA4B2ZNUETZUJST3U5MNDP5P46NA5CNFSM4H47LO3KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZC7FFQ#issuecomment-507900566",
"url": "https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/issues/21?email_source=notifications\u0026email_token=AFMA4B2ZNUETZUJST3U5MNDP5P46NA5CNFSM4H47LO3KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODZC7FFQ#issuecomment-507900566",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by chris1111** [comment]
stuck at boot screen that say nothing for me
Boot in verbose boot and post picture when it stuck
Also try different USB port and follow the guide and Video

**Comment by Mtkhan13** [comment]
Hi, I tried couple of things, like you said 
1- verbose mode it shows some errors, though it boots fine. but with out it it stuck at boot screen. (See attached images)
2- When booted verbose mode, GPU showed 3Mb (See attached image) and screen resolution was not right. I changed the resolution in plist. Though resolution seems ok now but not sure this was the good way to do it.
3- No Audio, Bluetooth, Wifi Or LAN working.  (See attached images)

I'm can also share Original DSDT extracted from Ubuntu tonight.


![WhatsApp Image 2019-07-03 at 10 20 53 PM](https://user-images.githubusercontent.com/22547975/60612082-51f93600-9de1-11e9-9b33-9123fbecad89.jpeg)
![WhatsApp Image 2019-07-03 at 10 20 54 PM](https://user-images.githubusercontent.com/22547975/60612083-5291cc80-9de1-11e9-8aa7-33e1020c2aef.jpeg)

![WhatsApp Image 2019-07-03 at 10 20 55 PM](https://user-images.githubusercontent.com/22547975/60612031-3130e080-9de1-11e9-855f-583275d03a93.jpeg)
![WhatsApp Image 2019-07-03 at 10 20 54 PM(1)](https://user-images.githubusercontent.com/22547975/60612040-3857ee80-9de1-11e9-8383-f0f4923cb760.jpeg)
